### PR TITLE
Add license and trivial README

### PR DESCRIPTION
**Pull request by dfinkel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/vimeo/go-clocks)](https://pkg.go.dev/github.com/vimeo/go-clocks)
```
Comment (sachinagada) [inline README.md:2]: nit: Could we add the tests passing badge here as well?

**Review by sachinagada — APPROVED** [review]
Just a small suggestion

File: README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+only covers a fraction of the `time` package's interface, and only covers the
+most trivial cases.
+
+The `Clock` interface in `go-clocks` covers both querying and waiting for time.
+
+### `FakeClock` and tests
+
+While `Clock` is a nice interface with both context-aware `SleepFor` and
+`SleepUntil`, it really shines when combined with the `FakeClock` in this
+repository. `FakeClock`
```
Comment (sachinagada) [inline README.md]: We can remove `FakeClock` at the end of this paragraph.

File: README.md (hunk 1 above)
Comment (dfinkel) [reply README.md:2]: Done

File: README.md (hunk 2 above)
Comment (dfinkel) [reply README.md]: Removed. And Updated all references to `FakeClock` to reference `fake.Clock`

**Pull request merged**